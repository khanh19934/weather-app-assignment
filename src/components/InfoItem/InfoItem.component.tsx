import React from 'react';

interface IProps {
  content: string;
  label: string;
}

const InfoItem: React.FC<IProps> = ({ content, label }: IProps) => {
  return (
    <div className='d-flex justify-content-between mb-2 info-item'>
      <span className='text-white'>{label}</span>
      <span className='text-white'>{content}</span>
    </div>
  );
};

export default InfoItem;
