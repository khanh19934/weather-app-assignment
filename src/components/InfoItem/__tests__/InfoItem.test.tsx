import React from 'react';
import renderer from 'react-test-renderer';
import InfoItem from '../InfoItem.component';

it('render correctly', () => {
  const tree = renderer.create(<InfoItem content='12%' label='HUMIDITY' />).toJSON();

  expect(tree).toMatchSnapshot();
});
