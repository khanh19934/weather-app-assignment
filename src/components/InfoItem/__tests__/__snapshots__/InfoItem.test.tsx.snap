// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="d-flex justify-content-between mb-2 info-item"
>
  <span
    className="text-white"
  >
    HUMIDITY
  </span>
  <span
    className="text-white"
  >
    12%
  </span>
</div>
`;
